src/themes/blueberry/index.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+
+import defaultTheme from '../../layout'
+import ExampleTypeConstructor from '../../exampleTypes/'
+
+import Input from './Input'
+import Checkbox from './Checkbox'
+import Fonts from './Fonts'
+import Style from './Style'
+import Table from './Table'
+import Button from './Button'
+import ButtonRadio from './ButtonRadio'
+import ListGroupItem from './ListGroupItem'
+import Wrap from './Wrap'
+import TagComponent from './TagComponent'
+import PagerItem from './PagerItem'
+import Adder from './Adder'
+
+export let Pager = props => (
+  <ExampleTypes.ResultPager Item={PagerItem} {...props} />
+)
+
+let baseTheme = {
+  ...defaultTheme,
```

Comment:
You went through the trouble of allowing partial themes, so no need to spread on the default theme here